certification-content-bundle-spec.md

```diff
@@ -0,0 +1,83 @@
+# Qwiklabs Certification Bundle Specification
+
+**Version 1**
+
+> This is a DRAFT document. We welcome feedback as this format evolves.
+
+## `qwiklabs.yaml` Structure
+
+Here's a sample `qwiklabs.yaml` file with all nested details removed to make it easier to see the general file structure.
+
+```yml
+entity_type: Certification
+schema_version: 1
+
+default_locale: en
+
+title: 
+  locales:
+    en: Virtual Machines in GCP
+
+description:
+  locales:
+    en: Prove what you know about VMs in GCP by doing these courses and exams!
+
+steps: ...
+```
+
+### Certification attributes
+
+attribute          | required | type       | notes
+-------------------| -------- | ---------- | -----------------------------------------
+default_locale     | ✓        | string     | Corresponds to the locale that the certification is authored in. Authoring tools can use this as a hint to notify localizers when content in the default locale is updated. Also, it provides a hint to the learner interface about which locale to display if an instruction/resource is not localized for the learner's current locale.
+schema_version     | ✓        | integer    | Which version of the certification bundle schema you are using
+title              | ✓        | dictionary | A locale dictionary of the certification title, such as "Virtual Machines in GCP"
+description        |          | dictionary | A locale dictionary of the certification description which describes the contents of the certification
+steps              | ✓        | array      | An array of `steps` (see [Steps](#steps) below for details)
+
+### Steps
+
+A certification is composed of multiple `steps`. A step can be either a Course Template or an exam that the user must complete to progress through the certification.
+
+#### Gated Steps
+
+Steps can optionally be marked as "gated". A gated step will be unavailable to the user until all preceding steps are completed, and all succeeding steps will be unavailable to the user until the gated step is complete.
```

Comment:
I think gating should only be before the step. So in a 5 step certification, if step 3 is marked gated, once user completes steps 1 and 2, steps 3, 4, and 5 become available.

Reply:
I guess the question is for the 'gate' step itself. Does it imply a single gate or separate enter/exit gates?